Code/Legacy/CrySystem/SystemInit.cpp

```diff
@@ -445,18 +441,6 @@ AZStd::unique_ptr<AZ::DynamicModuleHandle> CSystem::LoadDLL(const char* dllName)
         return handle;
     }
 
-    //////////////////////////////////////////////////////////////////////////
```

Comment:
nice, I will add a ticket to remove this function altogether, the only cry module left is CrySystem which is the one doing the loading.
there is another call to `LoadDLL` (`CSystem::InitVTuneProfiler` that can also be removed, but would remove all the `PROFILE_WITH_VTUNE` integration)
Created:
- https://github.com/o3de/o3de/issues/5261 to remove vtune integration in cry
- https://github.com/o3de/o3de/issues/5262 to remove all this dll loading functions from Cry